src/macros.rs

```diff
@@ -297,7 +297,7 @@ macro_rules! UNION {
     }) => (
         #[repr(C)] $(#[$attrs])* #[cfg(target_arch = "x86")]
         pub struct $name([$stype32; $ssize32]);
-        #[repr(C)] $(#[$attrs])* #[cfg(target_arch = "x86_64")]
+        #[repr(C)] $(#[$attrs])* #[cfg(any(target_pointer_width = "64"))]
```

Comment:
Why is this `any`?

Reply:
A remnant from mindlessly converting x86_64/aarch64 conditionals to use `target_pointer_width`.  I will fix.